src/request-pipeline/file-request.ts

```diff
@@ -5,34 +5,54 @@ import { EventEmitter } from 'events';
 import { parse } from 'url';
 import { MESSAGE, getText } from '../messages';
 import { stat, access } from '../utils/promisified-functions';
+import Asar from '../utils/asar';
 
 const DISK_RE: RegExp = /^\/[A-Za-z]:/;
 
-const TARGET_IS_NOT_FILE = 'The target of the operation is not a file';
+const TARGET_IS_NOT_FILE: string = 'The target of the operation is not a file';
+
+const asar = Asar.getInstance();
```

Comment:
Looks a bit overcomplicated, I think we can just create an `Asar`  instance here.